From cafe8a20c56b8a6e49c675ed4088da4ebf905df2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Motiejus=20Jak=C5=A1tys?= <motiejus.jakstys@spilgames.com>
Date: Sat, 6 Apr 2013 17:53:06 +0300
Subject: [PATCH 5/6] Add support for travis-ci service

This allows running tests on shared continuous integration testing
service travis-ci.org. Currently it does:
1. compile
2. run tests (incl. ones that require protobuffers)
---
 .travis.yml |    5 +++++
 travis-ci   |    9 +++++++++
 2 files changed, 14 insertions(+)
 create mode 100644 .travis.yml
 create mode 100755 travis-ci

diff --git a/.travis.yml b/.travis.yml
new file mode 100644
index 0000000..d8ccbf4
--- /dev/null
+++ b/.travis.yml
@@ -0,0 +1,5 @@
+language: c
+script: "./travis-ci"
+before_install:
+    - sudo apt-get install ocaml-nox camlp4-extra libfindlib-ocaml-dev
+    - sudo apt-get install libprotoc-dev protobuf-compiler
diff --git a/travis-ci b/travis-ci
new file mode 100755
index 0000000..ddedf24
--- /dev/null
+++ b/travis-ci
@@ -0,0 +1,9 @@
+#!/bin/sh -xe
+
+export OCAMLPATH=
+./configure --prefix=/usr
+make deps
+make
+
+. ./setenv.sh
+make -C tests
-- 
1.7.9.5

